<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graph_windows</class>
 <widget class="QMainWindow" name="graph_windows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>graph_windows</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="QPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>781</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Параметр</string>
    </property>
    <addaction name="Parametr_1"/>
    <addaction name="Parametr_2"/>
    <addaction name="Parametr_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Сеть</string>
    </property>
    <addaction name="Network_1"/>
    <addaction name="Network_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Parametr_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Первый</string>
   </property>
  </action>
  <action name="Parametr_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Второй</string>
   </property>
  </action>
  <action name="Parametr_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Третий</string>
   </property>
  </action>
  <action name="Network_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Считывать 10-12 байты</string>
   </property>
  </action>
  <action name="Network_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Считывать 13-14 байты</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
